ITC205_BaseLine_Code/src/Main.java

```diff
@@ -4,218 +4,211 @@
 
 public class Main {
 	
-	private static Scanner IN;
-	private static library LIB;
-	private static String MENU;
-	private static Calendar CAL;
-	private static SimpleDateFormat SDF;
+	private static Scanner takeInput;
+	private static library library;
+	private static String libraryMenu;
+	private static Calendar calendar;
+	private static SimpleDateFormat simpleDateFormat;
 	
 	
-	private static String Get_menu() {
-		StringBuilder sb = new StringBuilder();
+	private static String getMenu() {
+	    StringBuilder buildString = new StringBuilder();
 		

```

Comment:
D1 - delete tab in blank lines - lines 16, 35, 54, 56, 58, 60,  63, 65, 69, 73, 77, 81, 85, 89, 93, 97, 101, 105, 109, 114, 123, 189, 210, 215